code/src/routes/routes.tsx

```diff
@@ -17,6 +18,16 @@ const routes: RouteObject[] = [
       }
     ]
   },
+  {
+    path: "/products",
+    errorElement: <ErrorPage />,
+    children: [
+      {
+        element: <RootLayout />,
+        children: [{ index: true, element: <ProductsPage /> }]
+      }
+    ]
+  },
```

Comment:
I suggest creating object with routes and exporting it as const. For example
```
const routes = {
  home: "/",
  products: "/products",
  error ....
 } as const 
```

Reply:
Thanks for suggestion. I created routes file in `constants`.